.firebaserc

```diff
@@ -0,0 +1,5 @@
+{
+  "projects": {
+    "default": "eddiebot-acf4c"
```

Comment:
Just leaving a question here: maybe we could add some documentation on setting up Firebase locally?
I haven't tested locally the bot with this feature, but I'd assume it doesn't run unless I configure with my firebase project.

This is more of a question since newcomers might not know firebase, or know what is `firestore.rules` and `firestore.indexes.json` 😄

Reply:
Oh good point 👍 . The rules will be the same for everyone, but the project name will not be 🤦‍♂️ . Yes this needs more work in this PR to make things clear for people to run it locally themselves

Reply:
I added a DB structure diagram to the README. It is quite boring at the moment, but more data to come soon